gem/static/css/_enhanced.article-preview.scss

```diff
@@ -36,16 +36,32 @@
   }
 }
 
-.article-list {
-  display: flex;
-  flex-wrap: wrap;
+.article-list .article-preview  {
+  img {
+    width: 100%;
+  }
+  h3 {
+    margin: 0 0 9px 0;
+  }
+  p {
+    margin: 0;
+  }
+}
 
+.article-list  {
+  overflow: hidden;
 
+  .article-preview:nth-child(odd){
```

Comment:
Hmmm clear doesn't seem to work on each .article-preview but only on the third odd .article-preview item.